test/pod/run-test-pod

```diff
@@ -52,12 +52,22 @@ PHASE_RUNNING="Running"
 PHASE_SUCCEEDED="Succeeded"
 RETRY_INTERVAL=5
 
-# Wait until pod is running
+# Wait until pod is running or timeout
 echo "Waiting for test-pod to start runnning"
+TIMEOUT=90
+ELAPSED=0
 POD_PHASE=""
 until [ "${POD_PHASE}" == "${PHASE_RUNNING}" ]
 do
+    if [ "${ELAPSED}" -eq "${TIMEOUT}" ]; then
```

Comment:
`[ "${ELAPSED}" -ge "${TIMEOUT}" ]`?
use >=? I think it is possible that `ELAPSED` != `TIMEOUT` if `RETRY_INTERVAL` and `TIMEOUT` are not picked correctly.

Reply:
Good catch!